packages/components/package.json

```diff
@@ -60,6 +60,7 @@
 		"moment": "^2.22.1",
 		"re-resizable": "^6.4.0",
 		"react-colorful": "^5.3.0",
+		"react-confirm": "0.1.24",
```

Comment:
> I think it would be better to implement the functionality of react-confirm directly in our codebase (probably in the useConfirm hook), without importing this external package.
The implementation seems simple enough, and we may want to apply tweaks to it anyway.

Yeah, agreed. I'll look into it.

> It would also be good, IMO, to allow this component to be used declaratively (e.g. by adding the <Confirm /> tag in the JSX markup) instead of imperatively (i.e.by calling the confirm() function)

I think allowing providing both ways would be useful. The isolated rendering provided by the `react-confirm` approach is useful because it allows us to keep the API close to the existing `confirm`, and makes it trivial to migrate existing usage instances. I agree that we should also export the `Confirm` component and allow using it as is, to compose in any component's rendering tree.